azure-governance-cloudwise/azuredeploy.json

```diff
@@ -0,0 +1,573 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "skuName": {
```

Comment:
That  was our original plan, however a few customers who tried this @ Ignite suggested to keep these as parameters as they plan to configure it with a slightly better skus for their dev environment.